src/System/Relay/getMetaphysicsEndpoint.ts

```diff
@@ -0,0 +1,11 @@
+import { getENV } from "Utils/getENV"
+
+export const getMetaphysicsEndpoint = () => {
+  const APP_URL = getENV("APP_URL") ?? "http://localhost:4000"
```

Comment:
This should probably look at `PORT`

Reply:
We don't expose PORT to the client, not sure if that was deliberate or not. Thoughts?